plugins/services/src/js/structs/Application.js

```diff
@@ -65,7 +66,14 @@ module.exports = class Application extends Service {
    * @override
    */
   getImages() {
-    return FrameworkUtil.getServiceImages(this.getMetadata().images);
+    if (
```

Comment:
Can't we do something like this instead?
```js
const images = this.getMetadata().images || this.get("images");

return FrameworkUtil.getServiceImages(images);
```

Reply:
This won't work the same in some cases, for example if the metadata images are the same as the default ones.

Reply:
Is there an example when we have default images in the metadata but custom in `images`? If we have default images `getImageSizeFromImagesObject` won't return null, will it?

Reply:
If everyone can add to Universe, then such an example is possible.
`getImageSizeFromImagesObject` won't return null, but `getServiceImages` will return the default images.